src/main.js

```diff
@@ -8,10 +11,15 @@ var savedCovers = [
 var currentCover;
 
 // Add your event listeners here 👇
-
+window.addEventListener("load", randomizePoster);
 
 // Create your event handlers and other functions here 👇
-
+function randomizePoster(){
+  cover.src = covers[getRandomIndex(covers)],
+  title.innerText = titles[getRandomIndex(titles)],
+  firstTagline.innerText = descriptors[getRandomIndex(descriptors)],
+  secondTagline.innerText = descriptors[getRandomIndex(descriptors)]
```

Comment:
Question: Should we make sure that the two tag lines won't end up being the same word/ index number?

Reply:
Good point! Let's ask about that tomorrow.

Reply:
Sounds good! Aside from that, everything looks great!